src/pybind/mgr/cephadm/tests/test_scheduling.py

```diff
@@ -64,16 +55,16 @@ class NodeAssignmentTest(NamedTuple):
             ],
             ['host1', 'host2', 'host3']
         ),
-        # zero count + partial host list
+        # count 1 + partial host list
         NodeAssignmentTest(
             'mon',
-            PlacementSpec(count=0, hosts=['host3']),
+            PlacementSpec(count=1, hosts=['host3']),
             'host1 host2 host3'.split(),
             [
                 DaemonDescription('mon', 'a', 'host1'),
                 DaemonDescription('mon', 'b', 'host2'),
             ],
-            []
+            ['host3']
```

Comment:
why change this test case?

Reply:
cause, I'm getting a 
```python
ServiceSpecValidationError("num/count must be > 1")
``` error here.

Reply:
hmm, wondering why it was working before...

Reply:
we never called `validate()` for the placement spec! (cc @jschmid1 )